lib/reporter.js

```diff
@@ -225,6 +226,13 @@ class CucumberReporter {
         })
     }
 
+    getTitle (featureOrScenario) {
+        const name = featureOrScenario.getName()
+        const tags = featureOrScenario.getTags ? featureOrScenario.getTags() : []
```

Comment:
Tags could be in feature and scenario, so getTags should be available in both. Validation for safety reason (eg. in previous `getPayloadItem` was not mocked), if somehow method won't be available.